<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QuestDatabase</class>
 <widget class="QDialog" name="QuestDatabase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>849</width>
    <height>568</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>849</width>
    <height>568</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>849</width>
    <height>568</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Quest Database</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="minimumSize">
      <size>
       <width>819</width>
       <height>513</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>819</width>
       <height>513</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="infoTab">
      <attribute name="title">
       <string>Quest Information</string>
      </attribute>
      <widget class="QGroupBox" name="heroStateBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>52</y>
         <width>284</width>
         <height>401</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="title">
        <string>Hero Initial State</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <property name="sizeConstraint">
               <enum>QLayout::SetMinimumSize</enum>
              </property>
              <item>
               <widget class="QLabel" name="label_2">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>Maximum Health</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_3">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>Initial Health</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QSpinBox" name="maxHealthBox">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="maximum">
                 <number>16</number>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="initialHealthBox">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="maximum">
                 <number>16</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item row="2" column="0">
           <widget class="QTableView" name="abilityTable"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_4">
            <property name="font">
             <font>
              <pointsize>11</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Initial Ability Levels</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QLineEdit" name="questNameEdit">
       <property name="geometry">
        <rect>
         <x>102</x>
         <y>12</y>
         <width>704</width>
         <height>27</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>704</width>
         <height>27</height>
        </size>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>11</x>
         <y>12</y>
         <width>85</width>
         <height>27</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>85</width>
         <height>27</height>
        </size>
       </property>
       <property name="text">
        <string>Quest Name</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tilesetTab">
      <attribute name="title">
       <string>Tilesets</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <layout class="QFormLayout" name="formLayout_2">
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QListView" name="tilesetsList">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>217</width>
               <height>422</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>230</width>
               <height>422</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QPushButton" name="removeTilesetButton">
               <property name="minimumSize">
                <size>
                 <width>105</width>
                 <height>27</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>111</width>
                 <height>27</height>
                </size>
               </property>
               <property name="text">
                <string>Remove</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="addTilesetButton">
               <property name="minimumSize">
                <size>
                 <width>104</width>
                 <height>27</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>110</width>
                 <height>27</height>
                </size>
               </property>
               <property name="text">
                <string>Add...</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item row="0" column="1">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="setLabel">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>Current Tileset:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QGraphicsView" name="tilesetSelectedView">
             <property name="minimumSize">
              <size>
               <width>559</width>
               <height>436</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>720</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="OKButton">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
